giraffe/src/utils/PlotEnv.ts

```diff
@@ -118,10 +118,10 @@ export class PlotEnv {
   }
 
   public get xScale(): Scale<number, number> {
-    const getXScaleMemoized = this.fns.get('xScale', getLinearScale)
+    const scale = getScale(this.config.xScale)
```

Comment:
To memoize, it would be something like this:
<code>const getXScaleMemoized = this.fns.get(&#96;xScale_${this.config.xScale}&#96;, getScale(this.config.xScale))</code>